package.json

```diff
@@ -20,8 +20,12 @@
   },
   "homepage": "https://github.com/lulibrary/LAG-Utils#readme",
   "devDependencies": {
```

Comment:
There's no Dependency for the AWS-SDK that you require in the Topic class, can you add this as one.

Reply:
This is  fixed in https://github.com/lulibrary/LAG-Utils/pull/2/commits/502d786450479aefef2453f7086aa3459088afd1